code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -92,10 +92,11 @@
 	affect_ingest(M, alien, removed)
 
 /datum/reagent/nutriment/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
-	if(isvaurca(M))
-		M.adjustToxLoss(1.5 * removed)
-	if(alien && alien == IS_UNATHI)
-		return
+	if(alien)
```

Comment:
Why not just make this entire thing a switch?
```dm
switch (alien)
    if (IS_VAURCA)
        M.adjustToxLoss(1.5 * removed)
    if (IS_UNATHI)
        return
    else
        digest(M, removed)
```

Reply:
Vaurca can still digest nutrients.

Reply:
oh nvm you're right